test/cases/ast/_Float16.c

```diff
@@ -27,7 +27,7 @@ fn_def: 'fn (x: int, ...) void'
     var: '[1]struct __va_list_tag'
      name: va
 
-    builtin_call_expr: 'fn (d[1]struct __va_list_tag, ...) void'
+    builtin_call_expr: 'void'
```

Comment:
This was previously incorrect and showed the type of the builtin itself instead of the type of the return value